.travis.yml

```diff
@@ -1,37 +1,203 @@
-dist: trusty
+os: linux
 sudo: required
-group: edge
-jdk: oraclejdk8
-
-services:
-- docker
-script:
-- bash test/travis-bin/run-tests.sh
-- bash test/travis-bin/sonar.sh
-# Dockerhub password is inside travis settings
-- MODE=production bash build-docker-images.sh
+language: minimal
+
+env:
+  global:
+    secure: KWhqHGlzE1ioGyZzcWxoTdhJdSnXvORHIZXvHC7WRdnOLa14gTdRkdjezzVT2DIeHcqH5x5uffAaWHgXWYPzqgm64XpR86UAcQeqf5FxF7V91M0jIaBlktzTK7Xf8PGozh0hqJqD4lTcG4hLW99a/dI0Z/LULape5dNBZWHKGwk6XjYY7rQErtSw1l4jrZD2w6xEVjXQSckwFVnbF5whoYiwOJwYzyNnfrY6NNWHtTwzSKmdK1DUQr6Vk+eA+ggndWXe2GA3bJGM9K4lNbG/TcTLUTULOfkqwoZVejOrd5dpuNZqa/iRPubtVuFUZ0W0+DFGl8KBctdkbhqI4qk27uQPIVQO7zF36hR1oaQuSNXyXwWOVtVm3+ymu1YAfzjVUTiRBHg5Uda0Ink8yLo8Wpnp0gRq5Y3iVhymMYbiKHuzs52tnHfZ6NFjIH1M86uZhzrJhCqFKKCX9dzguj1PrM+7zYC0aXIxi3Yt6k1C2DY/8A+g201tBhMB7EwppilH6Rm2sqBXpUKI/P/LW/lJOUmY4EGBOPv+RGX0W3Av28NTI0PisCiNKgD8w5r+4l16/AhMBY3lmvTjZFd6IpfcJohWA25k2h7b4Opemu7/po7faMGMhqUTW64D98YhqiWtR0QpVtIRnLiol4GbwwUjT83p1Nw+korROXsei5ygX/w=
```

Comment:
Could you add a comment to explain what is the secure variable ?

Reply:
Oh I didn't notice, maybe SonarQube ?

Reply:
Maybe but I prefer to be sure :smile:. So let forget it this time and I'll perform a manual `travis.yml` linting on all our repos. Unfortunately this is not the only secured variable without comment... :disappointed: